contrib/kube-prometheus/manifests/0prometheus-operator-serviceMonitor.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
 spec:
   endpoints:
   - port: http
+    honorLabels: true
```

Comment:
Our CI is failing due to this line. The kube-prometheus project vendors the Prometheus Operator project to generate these manifests. But it is pinned to the latest release of the Prometheus Operator. Thereby your patch will only be applied to the kube-prometheus project once we cut the next Prometheus Operator release.

You can already use your patch by updating the jsonnet lockfile in the kube-prometheus project to use the latest prometheus-operator `master` version.

Reply:
@mxinden  - bit confused, should this just stay as it is until you cut the new release (then the PR can be incorporated?)

Or did you actually want me to change `contrib/kube-prometheus/jsonnetfile.lock.json` to use `master` and push that (this didn't actually work for me locally anyway when I ran the tests).

Reply:
@nabadger I am sorry for the confusion. Thanks for bearing with me.

1. You would need to remove this change (`git checkout upstream/master -- contrib/kube-prometheus/manifests/0prometheus-operator-serviceMonitor.yaml && git add . && git commit && git push`). 
2. Our CI will go green and we merge
3. Once we cut a release of the Prometheus Operator, we will update the Prometheus Operator reference in the kube-prometheus stack which in turn brings exactly this line change.

I am sorry for the complications and the thereby resulting delay.